types/next-auth/client.d.ts

```diff
@@ -50,10 +50,16 @@ declare function signin(
     provider?: string,
     data?: GenericObject & {
         callbackUrl?: string;
+        redirect?: boolean;
     },
 ): Promise<void>;
 declare const signIn: typeof signin;
-declare function signout(data?: { callbackUrl?: string }): Promise<void>;
+declare function signout(
+    data?: { 
+        callbackUrl?: string; 
+        redirect?: boolean; 
```

Comment:
Let's [add a test](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/4bea1bd982be671edf210b7010aba4ce0f627bd1/types/next-auth/next-auth-tests.ts#L265) to verify this new option 🪴